<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_10">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>501</width>
     <height>63</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_22">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>물품명</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>수량</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLineEdit" name="lineEdit_13"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_17"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_38">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>461</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>필요물품의 추가 버늩 누르면 생성</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>581</width>
     <height>191</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_12">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>579</width>
      <height>189</height>
     </rect>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>851</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>물품테이블 불러옴</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
